docs/guides/chip_tool_guide.md

```diff
@@ -1391,3 +1391,180 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
     ```
     ./chip-tool any read-by-id 0xFFFFFFFF 0xFFFFFFFF 1 0xFFFF
     ```
+
+<hr>
+
+## Saving users and credentials on door lock devices
+
+Matter door lock devices can store pools of users and credentials that allow you to configure different access scenarios. Each user and credential in the pool has an index value. Additionally, each user has a list of Occupied Credentials.
+
+By default, each door lock device comes with no user or credential defined, but it reserves several slots in both pools that can be populated with new users and credentials, up to the value specified in the the `NumberOfTotalUsersSupported` attribute and the `NumberOfCredentialsSupportedPerUser` attribute, respectively.
+
+For security reasons, all communication between users and credentials happens only using their respective index values. No other information is shared between both pools.
+
+The CHIP Tool lets you add users and credentials on door lock devices and securely match their indexes to one another. This is an optional feature available only when working with devices that implement the `doorlock` cluster.
+
+> **Note:**
+> Users and credentials can only be modified by whoever has the right permissions, as specified in the Access Control List.
+
+Complete the following steps:
```

Comment:
This sentence looks like taken out of context. Maybe it is worth adding something like: `To save credentials and users complete the following steps.`